.circleci/config.yml

```diff
@@ -5,26 +5,28 @@ jobs:
   build:
     docker:
       # Specify the Ruby version you desire here
-      - image: circleci/ruby:2.5.0
+      - image: circleci/ruby:2.5.3-node
```

Comment:
Switching the base image to Ruby 2.5.3 includes a more up-to-date version of `npm`, which enables `npm ci`. Can also install it manually during the build process, if we’d rather stick with Ruby 2.5.0.